web/config.py

```diff
@@ -10,6 +10,7 @@
 
 _TEMP_FOLDER_OBJECT = TemporaryDirectory()
 
+DEMO = False
```

Comment:
Would it be possible to move the demo flag to be a command line argument in `run.py` so that we don't have to hard code the website to be in demo mode?